source/assets/sass/components/_link-header.scss

```diff
@@ -60,7 +57,7 @@ h6 {
 
 .signature {
   a.anchor {
-    margin-left: -2.5rem;
+    margin-left: var(--gutter-double-sesqui-negative);
```

Comment:
"sesqui" 😍

Reply:
@jgerigmeyer I remember we used this named size on another project, but I can't remember which one.